.rubocop.yml

```diff
@@ -286,4 +292,7 @@ Lint/UselessAccessModifier:
 Lint/IneffectiveAccessModifier:
   Enabled: false
 
+Lint/AmbiguousBlockAssociation:
+  Enabled: false
+
```

Comment:
Have you looked into these new cops? Are any of them worth considering enabling in the future?